src/System.Net.Security/src/System/Net/SslStreamPal.Unix.cs

```diff
@@ -75,19 +75,9 @@ public static void QueryContextStreamSizes(SafeDeleteContext securityContext, ou
             streamSizes = new StreamSizes();
         }
 
-        public static int QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
+        public static void QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
```

Comment:
This is a PAL function, so the signature needs to match Windows. (Windows currently has a void return). If we did this we would need to update the Windows PAL function and the call site.  (Not saying that we can't do that though...)